libs/core/src/lib/select/select.component.ts

```diff
@@ -142,98 +171,160 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnInit, OnD
     @Output()
     readonly valueChange: EventEmitter<any> = new EventEmitter<any>();
 
+    /** @hidden */
+    @ContentChildren(OptionComponent, {descendants: true})
+    options: QueryList<OptionComponent>;
+
+    /** @hidden */
+    @ViewChild('selectControl')
+    controlElementRef: ElementRef;
+
+    /** Reference to root element for the mobile mode dialog */
+    @ViewChild('dialogContainer')
+    dialogContainerElementRef: ElementRef;
+
+    /** Reference to element containing list of options */
+    @ViewChild('selectOptionsListTemplate')
+    selectOptionsListTemplate: TemplateRef<any>;
+
     /** @hidden */
     calculatedMaxHeight: number;
 
     /** Current selected option component reference. */
-    private _selected: OptionComponent;
-
-    /** Subject triggered when the component is destroyed. */
-    private readonly _destroy$: Subject<void> = new Subject<void>();
-
-    /** Observable triggered when an option has its selectedChange event fire. */
-    private readonly _optionsStatusChanges: Observable<OptionComponent> = defer(() => {
-        const options = this.options;
-        if (options) {
-            return options.changes.pipe(
-                startWith(options),
-                switchMap(() => merge(...options.map((option) => option.selectedChange)))
-            );
-        }
-    }) as Observable<OptionComponent>;
+    selected: OptionComponent;
+
+    /** @hidden */
+    optionsArray: OptionComponent[];
```

Comment:
`options: OptionComponent[];`

optionsArray name is redundant because you can have many options and an array is a type